packages/suite/src/components/wallet/CoinjoinSummary/CoinjoinStatus.tsx

```diff
@@ -130,18 +132,16 @@ export const CoinjoinStatus = ({
     restoreSession,
     stopSession,
 }: CoinjoinStatusProps) => {
-    const isPaused = !!session.paused;
     const [isLoading, setIsLoading] = useState(false);
     const [isWheelHovered, setIsWheelHovered] = useState(false);
 
     const menuRef = useRef<HTMLUListElement & { close: () => void }>(null);
     const theme = useTheme();
 
-    const timeLeft = `${
-        (session.maxRounds - session.signedRounds.length) *
-        getEstimatedTimePerRound(!!session.skipRounds)
-    }h`;
-    const progress = session.signedRounds.length / (session.maxRounds / 100);
+    const { maxRounds, signedRounds, paused, sessionDeadline, phase } = session;
+
+    const progress = signedRounds.length / (maxRounds / 100);
```

Comment:
candidate for shared util. also can be found here packages/suite/src/components/suite/CoinjoinStatusBar.tsx

Reply:
done